src/mapping/tests/RadialGeoMultiscaleMappingTest.cpp

```diff
@@ -0,0 +1,124 @@
+#include <Eigen/Core>
+#include <algorithm>
+#include <memory>
+#include "logging/LogMacros.hpp"
+#include "mapping/Mapping.hpp"
+#include "mapping/RadialGeoMultiscaleMapping.hpp"
+#include "math/constants.hpp"
+#include "mesh/Data.hpp"
+#include "mesh/Mesh.hpp"
+#include "mesh/SharedPointer.hpp"
+#include "mesh/Utils.hpp"
+#include "mesh/Vertex.hpp"
+#include "testing/TestContext.hpp"
+#include "testing/Testing.hpp"
+
+using namespace precice;
+using namespace precice::mesh;
+
+BOOST_AUTO_TEST_SUITE(MappingTests)
+BOOST_AUTO_TEST_SUITE(RadialGeoMultiscaleMapping)
+
+BOOST_AUTO_TEST_CASE(testConsistentSpread)
+{
+  /*  The following test works by creating two dimensionally heterogeneous meshes, namely 1D and 3D, that intersect along the x-axis.
+      Then, the data is mapped from the vertices of the 1D mesh to defined vertices of the 3D mesh (hence, SPREAD).
+      The defined vertices are close to a 1D vertex and therefore we can predict which value from the 1D vertex should be assigned.
+      Finally, this expected behavior is tested.  
+  */
+
+  PRECICE_TEST(1_rank);
```

Comment:
Does the mapping currently work in parallel?
If yes -> add a test.
If not -> create an issue in the project.

Reply:
I think they don't work in parallel straight out of the box, so I'll create an issue.